templates/tfengine/components/cicd/manual/configs/tf-validate.yaml

```diff
@@ -15,36 +15,36 @@
 timeout: 600s
 
 steps:
-  - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:02b06198f1da423183937b60493bdaa20dedf36b1a852a1d7fbb5a537fd943fd"
+  - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:47db3e958fbaa0d95881cf99501ebf4522a261ddc68d9566b7cf70e26cf7cddb"
```

Comment:
Since this is a component and templated, I would recommend doing something like this here and in the other files:
```
# {{$cftContainer := "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:47db3e958fbaa0d95881cf99501ebf4522a261ddc68d9566b7cf70e26cf7cddb"}}
...
  - name : "{{$cftContainer}}"
```

Reply:
Maybe don't put a # because you will just see a random # in the final output

Reply:
Good point! Done.